pkg/apply/prune/prune.go

```diff
@@ -235,17 +235,13 @@ func (p *Pruner) removeInventoryAnnotation(ctx context.Context, obj *unstructure
 // if one occurs.
 func (p *Pruner) GetPruneObjs(
 	ctx context.Context,
-	inv inventory.Info,
+	inv inventory.Inventory,
 	objs object.UnstructuredSet,
 	opts Options,
 ) (object.UnstructuredSet, error) {
 	ids := object.UnstructuredSetToObjMetadataSet(objs)
-	invIDs, err := p.InvClient.GetClusterObjs(ctx, inv)
-	if err != nil {
-		return nil, err
-	}
 	// only return objects that were in the inventory but not in the object set
-	ids = invIDs.Diff(ids)
+	ids = inv.Objects().Diff(ids)
```

Comment:
Occurs to me while reading this that the Objects method actually returns ObjectIDs/ObjectRefs/ObjectMetas, not the objects themselves. It might be slightly easier to read if we rename the getter/setter.

Reply:
Done.